### ISSUE TITLE
macOS: Put .vpinball .pinmame configs into ~/Library/Application Support/VPinball

### ISSUE THREAD

**Issue by M-A-Boucher** [vpinball/vpinball#1987]
Currently, these folders are being created in ~/ ... however, they should technically be stored in Library/Application Support/VPinball or ~/Library/Application Support/VPinball (the latter is best for this type of app).

**Comment by toxieainc** [comment]
@jsm174 I guess this was changed in the meantime??

**Comment by jsm174** [comment]
Yes, this is now completed, see [FileLayout.md](https://github.com/vpinball/vpinball/blob/master/docs/FileLayout.md).

MacOS now stores in: `/Users/xxx/Library/Application Support/VPinballX/10.8/VPinballX.ini`
